cli/src/command.rs

```diff
@@ -657,18 +660,20 @@ pub fn run() -> Result<()> {
 				<E as NativeExecutionDispatch>::ExtendHostFunctions,
 			>;
 
-			use sc_service::TaskManager;
 			let registry = &runner.config().prometheus_config.as_ref().map(|cfg| &cfg.registry);
 			let task_manager = TaskManager::new(runner.config().tokio_handle.clone(), *registry)
 				.map_err(|e| Error::SubstrateService(sc_service::Error::Prometheus(e)))?;
 
+			let info_provider = timestamp_with_babe_info(6000);
+
 			ensure_dev(chain_spec).map_err(Error::Other)?;
 
 			#[cfg(feature = "kusama-native")]
 			if chain_spec.is_kusama() {
 				return runner.async_run(|_| {
 					Ok((
-						cmd.run::<service::kusama_runtime::Block, HostFunctionsOf<service::KusamaExecutorDispatch>>(
+						cmd.run::<service::kusama_runtime::Block, HostFunctionsOf<service::KusamaExecutorDispatch>, _>(
+							Some(info_provider)
```

Comment:
```suggestion
						cmd.run::<service::kusama_runtime::Block, HostFunctionsOf<service::KusamaExecutorDispatch>, _>(
							Some(timestamp_with_babe_info(service::kusama_runtime::MILLISECS_PER_BLOCK))
```
and similarly others.

Reply:
great! thank you for this suggestion - first time I didn't realize that I have access to these parameters; the suggestion is now applied